src/Microsoft.AspNetCore.OData/Routing/ODataActionSelector.cs

```diff
@@ -80,10 +81,23 @@ public ActionDescriptor SelectBestCandidate(RouteContext context, IReadOnlyList<
         {
             RouteData routeData = context.RouteData;
             ODataPath odataPath = context.HttpContext.ODataFeature().Path;
+
+            var routers = routeData.Routers;
+            var odataRoute = routeData.Routers.FirstOrDefault((router) =>
```

Comment:
>routeData.Routers [](start = 29, length = 17)

it's basic scenario to register multiple ODataRoute in the application. It's supported in Web API OData.

Reply:
Thanks for the clarification.